glean-core/android/src/test/java/mozilla/telemetry/glean/acmigration/GleanDataMigrationTest.kt

```diff
@@ -48,33 +87,51 @@ class GleanDataMigrationTest {
 
         val context = ApplicationProvider.getApplicationContext<Context>()
 
+        // Record some pending metrics data so that a 'metrics' ping can get
+        // sent.
+        addRecordedPendingMetricsData(context)
+
         // Make sure we did not migrate so that a new migration process starts.
         val migrator = GleanACDataMigrator(context)
         migrator.testResetMigrationStatus()
 
         // Set a fake sequence number for the baseline ping.
         setFakeSequenceNumber(context, "baseline", 37)
+        setMetricsPingLastDate(context, "2015-06-11T12:36:00-06:00")
 
         // Start Glean and point it to a local ping server.
+        // This does not clear any previously stored data.
         Glean.resetGlean(
             context,
             Configuration().copy(
                 serverEndpoint = "http://" + pingServer.hostName + ":" + pingServer.port,
                 logPings = true
             ),
-            true
+            false
         )
 
         // Trigger a baseline and a metrics ping.
         Pings.baseline.send()
         triggerWorkManager()
 
+        val requests = mutableMapOf<String, String>()
+        for (i in 0..1) {
+            val request = pingServer.takeRequest(20L, TimeUnit.SECONDS)
+            val docType = request.path.split("/")[3]
+            requests[docType] = request.body.readUtf8()
+        }
+
+        // Check that we received the expected sequence number for the baseline ping.
+        val baselineJson = JSONObject(requests["baseline"]!!)
+        assertEquals("baseline", baselineJson.getJSONObject("ping_info")["ping_type"])
+        assertEquals(37, baselineJson.getJSONObject("ping_info")["seq"])
+/*
```

Comment:
Just curious, but did you mean to leave this commented out code here?

Reply:
No, sorry about that. It was me trying to write an integration test for this. It's not simple and requires messing with lots of internals, so I left it out.